src/lib/viewers/controls/annotations/__tests__/AnnotationsControls-test.tsx

```diff
@@ -18,10 +18,14 @@ describe('AnnotationsControls', () => {
         let unmount = (): void => {
             // placeholder
         };
+        let found = false;
 
         beforeEach(() => {
             jest.spyOn(React, 'useEffect').mockImplementation(cb => {
-                unmount = cb() as () => void; // Enzyme unmount helper does not currently invoke useEffect cleanup
+                if (!found) {
```

Comment:
After adding `TargetedClickThroughGuideTooltip` there is now more than one useEffect spied on. The imported useEffect from BUIE that is being spied on is https://github.com/box/box-ui-elements/blob/master/src/features/targeting/utils/useOnClickBody.js.

Reply:
Can we mock `AnnotationsTargetedTooltip`, instead? It doesn't look like it's under test at the moment.